docs/docs/networks/picasso/governance.md

```diff
@@ -1,28 +1,74 @@
+
+- Root
+- Whitelisted Caller (Fast Track)
+
+Additional tracks are also planned to be added during Phase 1 such as General Admin, Treasury and more.
+
+The governance model during Phase 1 will introduce four separate roles, each serving a unique purpose:
+
+- Proposal Creation: Enabling the ability to create proposals, fostering innovation and community-driven development.
+- Voting: Allowing GOV holders to vote on proposals, ensuring that decisions reflect the collective will of the community.
+- Canceling Proposals/Slashing: Implementing mechanisms to cancel proposals and penalize those who voted maliciously.
+- Expedition: Introducing measures to speed up the voting process when needed, ensuring timely decision-making without compromising on democratic principles.
+
+These tracks and origins are designed to ensure a balanced and fair approach, aligning with our commitment to transparency, decentralization, and community engagement.
+
+## OpenGov Parameters
+
+The following two tables provide information about the voting and decision-making processes for the two collectives. They summarize the support thresholds required for various actions, the time it takes for those actions to pass when specific support percentages are met, and the confirmation and decision periods for each track. With X % of support, referenda can pass after Y duration (times in the table) since the beginning of referenda if the approval rate is above the approval curve. The decision deposit for Root is 500,000 PICA whereas for Whitelist, its is 50,000 PICA. 
+
+
+| Support Threshold | Whitelist Time (Passing) | Root Time (Passing) |
+|-------------------|--------------------------|----------------------|
+| 10%               | 12 hours        | 5 days 14 hours      |
+| 20%               | 3 hours 20 minutes       | 4 days 4 hours       |
+| 30%               | 1 hour 15 minutes        | 2 days 18 hours      |
+| 10%               | 12 hours 0 minutes       | 5 days 14 hours      |
+| 20%               | 3 hours 20 minutes       | 4 days 4 hours       |
+| 30%               | 1 hour 15 minutes        | 2 days 18 hours      |
+
+
+| Track Type            | Confirm Period    | Decision Period (Voting) |
+|-----------------------|-------------------|--------------------------|
+| Whitelist        | 30 minutes        | 4 days                   |
+| Root             | 1 day             | 7 days                   |
+
+The following table summarizes the rules and requirements related to certain actions within the OpenGov system. It provides a concise reference for understanding the conditions and costs associated with various actions in the OpenGov process.
+
+
+| Action        | Origin & Threshold         | Submission Deposit | 
+|---------------|----------------------------|--------------------|
+| Submit        | Anyone from any collective | 1 PICA             | 
+| Cancel        | 1/3 Tech Committee        | 1 PICA             | 
```

Comment:
cancel and kill doesnt have submission deposit. only submission, remove this column and add a sentence about it